easybuild/easyconfigs/s/slepc4py/slepc4py-3.7.0-CrayIntel-17.08-complex-64.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'slepc4py'
+version = '3.7.0'
+
+py_maj_ver = '3'
+py_min_ver = '6'
+py_rev_ver = '1.1'
+
+pyver = '%s.%s.%s' % (py_maj_ver, py_min_ver, py_rev_ver)
+pyshortver = '%s.%s' % (py_maj_ver, py_min_ver)
+
+versionsuffix = '-complex-64'
+
+homepage = 'http://slepc.upv.es/'
```

Comment:
Are you sure this is part of the official slepc project? If not, could you please update the homepage and description? I found this on the web https://bitbucket.org/slepc/slepc4py

Reply:
fixed!